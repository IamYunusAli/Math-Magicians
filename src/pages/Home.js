import React from 'react';
import styles from './pages.module.css';

const Home = () => (
  <div className={styles.home}>
    <h2>welcome to homepage</h2>
    <p>
      Mathemagician is a neologism, specifically a portmanteau,
      that combines mathematician and magician. A great number of
      self-working mentalism tricks rely on mathematical principles.
      Max Maven often utilizes this type of magic in his performance.
    </p>
  </div>
);

export default Home;
