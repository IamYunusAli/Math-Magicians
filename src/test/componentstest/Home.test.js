import renderer from 'react-test-renderer';
import Home from '../../pages/Home';

test('should match Home snapshot', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
