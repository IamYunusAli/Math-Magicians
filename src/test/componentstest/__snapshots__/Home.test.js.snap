// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Home snapshot 1`] = `
<div
  className="home"
>
  <h2>
    welcome to homepage
  </h2>
  <p>
    Mathemagician is a neologism, specifically a portmanteau, that combines mathematician and magician. A great number of self-working mentalism tricks rely on mathematical principles. Max Maven often utilizes this type of magic in his performance.
  </p>
</div>
`;
